recipes/_master_slurm_config.rb

```diff
@@ -29,6 +29,13 @@
   mode '0755'
 end
 
+# Create directory configured as StateSaveLocation
+directory '/var/spool/slurm.state' do
```

Comment:
Why don't use the default folder `/var/spool` as mentioned in the [documentation](https://slurm.schedmd.com/slurm.conf.html)?

Reply:
Or another idea could be to use: `/var/spool/slurmctld` since it's specific for the `slurmctld` daemon.

Reply:
For this one I'd rather have a subdirectory reserved to the slurm user